travis/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# No test data, then remove submodule (git automatically clones recursively)
+if [ ${TEST} == none ];
+ then git submodule deinit -f .;
+fi
+
+# Install test data as a package, then remove the submodule
+if [ ${TEST} == package ];
+ then pip install ./sherpa-test-data;
+ git submodule deinit -f .;
+fi
+
+# Build smoke test switches, to ensure requested dependencies are reachable
+if [ -n "${XSPECVER}" ]; then XSPECTEST="-x -d"; fi
+if [ -n "${FITS}" ] ; then FITSTEST="-f ${FITS}"; fi
+smokevars="${XSPECTEST} ${FITSTEST} -v 3"
+
+# Install coverage tooling and run tests using setuptools
+if [ ${TEST} == submodule ]; then
+    pip install pytest-cov; python setup.py -q test -a "--cov sherpa --cov-report term";
+fi
+
+# Run smoke test
+cd /home;
+sherpa_smoke ${smokevars};
+
+# Run regression tests using sherpa_test
+if [ ${TEST} == package ] || [ ${TEST} == none ];
+    then cd $HOME;
+    sherpa_test;
+fi
```

Comment:
github is complaining about a lack of a new line here; do we care about that?

Reply:
Yeah, we should please our masters!